app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Admin::UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
please create a before_actioin named `set_user` and use it where @user is required,